From 9471dbadf6d6e7633cf74da79efb5e60782bb5ec Mon Sep 17 00:00:00 2001
From: Anders Roxell <anders.roxell@linaro.org>
Date: Tue, 4 Jun 2019 08:58:26 +0200
Subject: [PATCH 6/6] zsmalloc: remove unused variable
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The variable 'entry' is no longer used and the compiler rightly
complains that it should be removed.

../mm/zsmalloc.c: In function ‘zs_pool_stat_create’:
../mm/zsmalloc.c:648:17: warning: unused variable ‘entry’ [-Wunused-variable]
  struct dentry *entry;
                 ^~~~~

Rework to remove the unused variable.

Fixes: 4268509a36a7 ("zsmalloc: no need to check return value of debugfs_create functions")
Signed-off-by: Anders Roxell <anders.roxell@linaro.org>
(am from https://lore.kernel.org/lkml/20190604065826.26064-1-anders.roxell@linaro.org/)
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 mm/zsmalloc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mm/zsmalloc.c b/mm/zsmalloc.c
index 1347d7922ea2..db09eb3669c5 100644
--- a/mm/zsmalloc.c
+++ b/mm/zsmalloc.c
@@ -645,8 +645,6 @@ DEFINE_SHOW_ATTRIBUTE(zs_stats_size);
 
 static void zs_pool_stat_create(struct zs_pool *pool, const char *name)
 {
-	struct dentry *entry;
-
 	if (!zs_stat_root) {
 		pr_warn("no root stat dir, not creating <%s> stat dir\n", name);
 		return;
-- 
2.22.0.rc3

